utils/data-processing-initialization/src/main/java/com/github/cafdataprocessing/utilities/initialization/WorkflowInitializer.java

```diff
@@ -54,19 +54,15 @@
 
     private ActionTypeNameResolver actionTypeNameResolver;
     private final BoilerplateNameResolver boilerplateNameResolver;
-    private final ClassificationWorkflowNameResolver classificationWorkflowNameResolver;
 
     /**
      * Create instance of WorkflowInitializer with provided parameters.
      * @param processingApiUrl URL of processing API to contact during initialization.
      * @param boilerplateNameResolver for use in resolving expression/tag names to IDs in workflow.
      * @param actionTypeNameResolver for use in resolving action type names to IDs in workflow.
-     * @param classificationWorkflowNameResolver for use in resolving classification workflow names to IDs in processing
-     *                                           workflow.
      */
     public WorkflowInitializer(String processingApiUrl, BoilerplateNameResolver boilerplateNameResolver,
-                               ActionTypeNameResolver actionTypeNameResolver,
-                               ClassificationWorkflowNameResolver classificationWorkflowNameResolver){
+                               ActionTypeNameResolver actionTypeNameResolver){
         final ApiClient apiClient = new ApiClient();
```

Comment:
This is a breaking change for Darwin integration point. We would need to increase the version number for this project accordingly.
https://github.houston.softwaregrp.net/Verity/common-lib/blob/b47139d4380813a48888f00c3410b74d23320dc1/aspen-common/policy-common/src/main/java/com/hpe/aspen/policy/common/PolicyWorkflowService.java#L108

Reply:
Good catch.  Let's take on the work to update the code that they have that uses this as well.